packages/shared/src/components/sidebar/DiscoverSection.tsx

```diff
@@ -20,8 +20,9 @@ export function DiscoverSection({
   ...defaultRenderSectionProps
 }: DiscoverSectionProps): ReactElement {
   const hasCommentFeed = useFeature(feature.commentFeed);
-  const { checkHasCompleted, completeAction } = useActions();
-  const hasCompletedCommentFeed = checkHasCompleted(ActionType.CommentFeed);
+  const { checkHasCompleted, completeAction, isActionsFetched } = useActions();
+  const hasCompletedCommentFeed =
```

Comment:
This now basically means it flashes the other way around (from not -> showing) but you don't notice really as it's on page load.

Reply:
I think issue is some conditions are meant to be true by default and some false, we can maybe add default value for `checkHasCompleted` so that in cases where we expect `true` by default we can say `checkHasCompleted(action, true)`?

Reply:
Yeah probably, lets log it for future 👍